pkg/oc/cli/cmd/observe/observe.go

```diff
@@ -716,6 +718,13 @@ func measureCommandDuration(m *prometheus.SummaryVec, fn func() error, labels ..
 		statusCode = -1
 	}
 	m.WithLabelValues(append(labels, strconv.Itoa(statusCode))...).Observe(float64(duration / time.Millisecond))
+
+	if err != nil && err.Error() == errNoChildProc {
```

Comment:
I'd rather not hardcode the value like that. See if you can get some reasonable information from the error, when being cast to [exec.ExitError](https://golang.org/pkg/os/exec/#ExitError). I think you should be able to get `ProcessState.Sys` which 'returns system-dependent exit information about the process. Convert it to the appropriate underlying type, such as syscall.WaitStatus on Unix, to access its contents. ' from there you should be able to get some system-wide constant.

Reply:
agree, this isn't a reasonable fix